composer.json

```diff
@@ -3,22 +3,26 @@
     "type": "neos-package",
     "description": "This package provides functionality for using Elasticsearch on top of Neos.ContentRepository.Search",
     "homepage": "http://flowpack.org/",
-    "license": [
-        "LGPL-3.0"
-    ],
+    "license": "LGPL-3.0",
     "require": {
         "php": ">=7.0",
-        "flowpack/elasticsearch": "~2.0 || dev-master",
-        "neos/content-repository-search": "~3.0 || dev-master",
-        "neos/flow": "~4.0 || dev-master",
-        "neos/fluid-adaptor": "~1.0 || dev-master"
+        "flowpack/elasticsearch": "^2.0",
+        "neos/content-repository": "^3.0",
+        "neos/content-repository-search": "^3.0",
+        "neos/eel": "^4.0",
+        "neos/flow": "^4.0",
+        "neos/fluid-adaptor": "^4.0",
+        "neos/utility-arrays": "^4.0"
```

Comment:
Some of those are required by Flow anyway, I'd not require them explicitly (again). After all, in Flow we  require only `doctrine/orm`, not all it's transitive dependencies…

But, I know there are differing views regarding that, so: was this done on purpose?